wurst/objects/units/ShopObjects.wurst

```diff
@@ -0,0 +1,36 @@
+package ShopObjects
+
+import Assets
+import UnitObjEditing
+
+import LocalObjectIDs
+
+public class ShopHeroDefinition extends HeroDefinition
+  construct(int newId, int origId)
+    super(newId, origId)
+    this
+      ..setLumberCost(0)
+      ..setFoodCost(0)
+      ..setGoldCost(0)
+      ..setStockStartDelay(0)
+      ..setStockReplenishInterval(999) // Will be programatically refilled
+      ..setStockMaximum(1)
+
+@compiletime function createDummies()
+  new ShopHeroDefinition(UNIT_RANDOM_TROLL, UnitIds.shadowhunter)
+    ..setName("Random Troll")
+    ..setTooltipBasic("Ran|c00ff00ffd|rom")
+    ..setTooltipExtended("")
+    ..setHotkey("D")
+    ..setStockReplenishInterval(0)
+    ..setIconGameInterface("ReplaceableTextures\\CommandButtons\\BTNShoveler.blp")
+    ..setUnitSoundSet("")
```

Comment:
Does this actually matter? The unit is removed upon entry, so it's not clear to me that any sounds would actually occur.

Reply:
The spawning sound theme was happening on my pc so i nulled it